cli/cmd/policy.go

```diff
@@ -442,9 +489,99 @@ func listPolicyTags(_ *cobra.Command, args []string) error {
 		return cli.OutputJSON(policyTagsResponse.Data)
 	}
 	if len(policyTagsResponse.Data) == 0 {
-		cli.OutputHuman("There were no policy tags found.")
+		cli.OutputHuman("There were no policy tags found.\n")
 		return nil
 	}
 	cli.OutputHuman(renderSimpleTable([]string{"Tag"}, policyTagsTable(policyTagsResponse.Data)))
 	return nil
 }
+
+func disablePolicy(_ *cobra.Command, args []string) error {
+	state := false
```

Comment:
Why is this variable here? what is it used for?

Reply:
UpdatePolicy enabled is a pointer 
Enabled       *bool
api/policy.go:66